### PR TITLE
feat: [245] Update user signup success and error messages

### PR DESCRIPTION
**Pull request by mmnoo**
after a suer signs up successfully, they will see a toast  and message on the login page they are navigated to saying to check their email. 

to test:
 - go to the sign up page
 - sign up
 - see the toast and message on the login page when redirected
 
 
Screen Shot 2022-09-16 at 12 48 23 PM

### CODE REVIEW

**Comment by julia-rom** [comment]
This works, something else that could improve the user flow is disabling the sign up button after it is clicked and adding some sort of indictor (ex: verifying, signing up, etc)

File: mrtt-ui/src/language.js [hunk 1]
```diff
@@ -2,14 +2,16 @@ const error = {
...
+  generic: 'Something went wrong. Please try again.',
   submit: 'Saving failed. Please try again.'
 }
 
 const success = {
-  submit: 'The data has been saved.',
-  getEditThingSuccessMessage: (thing) => `${thing} has been edited.`,
   getCreateThingSuccessMessage: (thing) => `${thing}, has been created.`,
-  getDeleteThingSuccessMessage: (thing) => `${thing} has been deleted.`
+  getDeleteThingSuccessMessage: (thing) => `${thing} has been deleted.`,
```
Comment (julia-rom) [inline mrtt-ui/src/language.js:11]: what is thing? can you be more specific?

**Review by julia-rom — COMMENTED** [review]
ideally we have some sort of submitting state after a user is signing up, and also disable the sign up button while this is happening

File: mrtt-ui/src/language.js (hunk 1 above)
Comment (mmnoo) [reply mrtt-ui/src/language.js:11]: I just added the comma. But thing is the thing/noun being deleted

Comment (mmnoo) [reply mrtt-ui/src/language.js:11]: agree about submitting state, but thats scope creep for this ticket. Feel free to make another ticket and we can priortize it as a team in the next meeting.

Comment (julia-rom) [reply mrtt-ui/src/language.js:11]: [added issue here](https://github.com/globalmangrovewatch/gmw-users/issues/308)

**Pull request merged**